lib/teiserver/battle/libs/balance_lib.ex

```diff
@@ -374,13 +395,7 @@ defmodule Teiserver.Battle.BalanceLib do
 
           {team_id, players} ->
             top_player =
-              players
-              |> Enum.map(fn userid ->
-                {ratings[userid], userid}
```

Comment:
This is a bug hence I had to remove and replace with a different function for captains. ratings is a variable defined above as tuple of team ids and team ratings. 
```
ratings[userid]
```
is completely wrong.